daemon/container.go

```diff
@@ -649,12 +637,11 @@ func (container *Container) ReleaseNetwork() {
 	if container.Config.NetworkDisabled || !container.hostConfig.NetworkMode.IsPrivate() {
 		return
 	}
-	eng := container.daemon.eng
 
-	job := eng.Job("release_interface", container.ID)
-	job.SetenvBool("overrideShutdown", true)
-	job.Run()
-	container.NetworkSettings = &NetworkSettings{}
+	_ = bridge.Release(container.ID)
```

Comment:
I mean that this error not checked at all in code. No one uses it. Maybe better to log it inside function and not return err.
I thought about `Warn`